.ci.yaml

```diff
@@ -42,6 +42,7 @@ targets:
       add_recipes_cq: "true"
       script: device_doctor/tool/build.sh
       cipd_name: flutter/device_doctor/mac-amd64
+      device_type: none
```

Comment:
Don't you also have to change the platform for this to take effect? I have had this issue with emulators where it needs to run on a linux platform and not linux_android platform.

Reply:
Mac doesn't specify the device type here? so it should be ok?

Reply:
Ah yeah I see that. You are good.